clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -102,6 +102,14 @@ class CIRGenBuilderTy : public mlir::OpBuilder {
   // Attribute helpers
   // -----------------
   //
+
+  /// Get constant address of a global variable as an MLIR attribute.
+  mlir::Attribute getGlobalViewAttr(mlir::cir::GlobalOp globalOp) {
+    auto type = getPointerTo(globalOp.getSymType());
+    auto symRef = mlir::FlatSymbolRefAttr::get(globalOp.getSymNameAttr());
+    return mlir::cir::GlobalViewAttr::get(type, symRef);
```

Comment:
Sweet, can you replace existing uses of `GlobalViewAttr::get` to also use this interface if possible?